<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeerSlashProcWidgetForm</class>
 <widget class="QWidget" name="SeerSlashProcWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Process Browser</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="searchLineEdit">
     <property name="toolTip">
      <string>Search in the list of processes. &quot;*&quot; is allowed.</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Search...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="refreshToolButton">
     <property name="toolTip">
      <string>Refresh the list of processes.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="view-refresh"/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTreeWidget" name="processTreeWidget">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <column>
      <property name="text">
       <string>PID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Command Line</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
